web/server/codechecker_server/api/mass_store_run.py

```diff
@@ -1007,6 +1007,7 @@ def get_skip_handler(
                     return skiplist_handler.SkipListHandler(skip_content)
             except (IOError, OSError) as err:
                 LOG.warning("Failed to open skip file: %s", err)
+                raise
```

Comment:
I think the above log message can be change to error instead of warning.

Reply:
Done